tests/Unit/Builder/ListBuilderTest.php

```diff
@@ -58,14 +59,14 @@ class ListBuilderTest extends TestCase
 
     public function setUp()
     {
-        $this->guesser = $this->createMock('\Sonata\AdminBundle\Guesser\TypeGuesserInterface', [], []);
+        $this->guesser = $this->createMock(TypeGuesserInterface::class, [], []);
```

Comment:
IIRC, there are no more parameters for method: `createMock()`:

```php
    /**
     * Returns a test double for the specified class.
     *
     * @param string|string[] $originalClassName
     *
     * @throws Exception
     * @throws \InvalidArgumentException
     */
    protected function createMock($originalClassName): MockObject
    {
        return $this->getMockBuilder($originalClassName)
                    ->disableOriginalConstructor()
                    ->disableOriginalClone()
                    ->disableArgumentCloning()
                    ->disallowMockingUnknownTypes()
                    ->getMock();
    }
```

same for the following `createMock()`-calls